viewer-app/root.tsx

```diff
@@ -283,6 +284,7 @@ class Root extends React.Component<{}, {
 
         <ComponentSection title={'Images'}>
           <Image url="http://via.placeholder.com/350x150" style={testImageStyle} height={150} width={350}/>
+          <Avatar imgSrc="http://via.placeholder.com/150x150" height={40} width={40} circle={true} />
```

Comment:
does it scale the 150x150 to fit inside the 40x40 or does it overflow?

Reply:
scales to fit into 40X40

Reply:
height/width props take precedent over image url